public/video-ui/src/components/Header.jsx

```diff
@@ -85,8 +85,9 @@ export default class Header extends React.Component {
             );
           }}
         >
-          <option value="false">All videos</option>
-          <option value="true">Loops</option>
+          <option value="">All videos</option>
+          <option value="url">Loops</option>
```

Comment:
Any self-hosted video (platform is `url`) is currently a loop.

Reply:
Note this will lead to url query params which are slightly more opaque to users. I think this is okay; the dropdown will indicate they are searching for loops.